distributions/test/src/test/java/com/connexta/replication/distributions/test/IonReplicationE2E.java

```diff
@@ -0,0 +1,175 @@
+import static com.connexta.replication.distributions.test.TestUtils.SITE;
+import static com.connexta.replication.distributions.test.TestUtils.clearSolrCore;
+import static com.connexta.replication.distributions.test.TestUtils.createProductOnNode;
+import static com.connexta.replication.distributions.test.TestUtils.defaultReplicationContainer;
+import static com.connexta.replication.distributions.test.TestUtils.defaultSolrContainer;
+import static com.connexta.replication.distributions.test.TestUtils.deleteQuietly;
+import static com.connexta.replication.distributions.test.TestUtils.extractDateTimeAttribute;
+import static com.connexta.replication.distributions.test.TestUtils.extractStringAttribute;
+import static com.connexta.replication.distributions.test.TestUtils.generateTestId;
+import static com.connexta.replication.distributions.test.TestUtils.getById;
+import static com.connexta.replication.distributions.test.TestUtils.getFileAsStream;
+import static com.connexta.replication.distributions.test.TestUtils.getFileAsText;
+import static com.connexta.replication.distributions.test.TestUtils.nodeHasMetadata;
+import static com.connexta.replication.distributions.test.TestUtils.nodeHasResourceModifiedAfter;
+import static com.connexta.replication.distributions.test.TestUtils.postToSolrCore;
+import static com.connexta.replication.distributions.test.TestUtils.updateProductOnNode;
+import static com.connexta.replication.distributions.test.TestUtils.waitAtMost30SecondsUntil;
+import static org.apache.commons.lang3.StringUtils.containsIgnoreCase;
+import static org.junit.Assert.assertTrue;
+
+import com.jayway.restassured.RestAssured;
+import com.jayway.restassured.path.xml.XmlPath;
+import java.io.ByteArrayInputStream;
+import java.io.InputStream;
+import java.time.Instant;
+import java.util.HashSet;
+import java.util.Map;
+import java.util.Set;
+import org.junit.After;
+import org.junit.AfterClass;
+import org.junit.BeforeClass;
+import org.junit.ClassRule;
+import org.junit.Test;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.testcontainers.containers.GenericContainer;
+import org.testcontainers.containers.Network;
+import org.testcontainers.containers.output.Slf4jLogConsumer;
+
+public class IonReplicationE2E {
+  private static final Logger LOGGER = LoggerFactory.getLogger(IonReplicationE2E.class);
+
+  // used to collect the replication service logs and print them with the test logs
+  private static final Slf4jLogConsumer logConsumer = new Slf4jLogConsumer(LOGGER);
+
+  @ClassRule public static Network network = Network.newNetwork();
+
+  private static final String sourceUrl = System.getenv("REPLICATION_SRC");
+  private static final String destinationUrl = System.getenv("REPLICATION_DEST");
+
+  public static GenericContainer solr = defaultSolrContainer(network);
+
+  public static GenericContainer replication = defaultReplicationContainer(network, "Ion");
```

Comment:
This test class is the same as the Classic Tests with the exception of using the Ion profile and there's no deleted test case.